src/app/shell/personal-cabinet/applications/application-card/application-card.component.html

```diff
@@ -17,11 +17,11 @@ <h4 class="name title">{{ application.child.lastName }} {{ application.child.fir
         application?.workshop.title }} </h4>
       <div class="organization" fxLayout="row" fxLayoutAlign="center center" *ngIf="userRole === role.parent">
         <mat-icon class="text_icon">home</mat-icon>
-        <p class="text">{{application?.workshop.providerTitle}}</p>
+        <p class="text workshop-title">{{application?.workshop.providerTitle}}</p>
```

Comment:
add router link to workshop-details, add class text-link, remove class workshop-title